drive.ts

```diff
@@ -0,0 +1,58 @@
+namespace crickit {
+    const CRICKIT_PWM_MIN = 0;
+    const CRICKIT_PWM_MAX = 65535;
+
+    //% fixedInstances
+    export class Drive {
+        private _id: number;
+        private _pin: number;
+
+        constructor(id: number, pin: number) {
+            this._id = id;
+            this._pin = pin;
+        }
+
+        /**
+         * set the drive duty cycle
+         */
+        //% group="Drives"
+        //% weight=100
+        //% blockId=driveanalogwrite block="crickit set %pin value to %value=number"
```

Comment:
drop the =number

Reply:
we use "analog write ..." terminology in the CPX pins. we should align with that.
![image](https://user-images.githubusercontent.com/4175913/40732714-be9eaa26-63e8-11e8-8a15-fb046ca2e3de.png)